integrations/telegram/src/misc/utils.ts

```diff
@@ -1,8 +1,56 @@
-import { axios } from '@botpress/client'
+import { axios, Conversation } from '@botpress/client'
+import { AckFunction } from '@botpress/sdk'
 import { IntegrationLogger } from 'src'
-import { USER_PICTURE_MAX_SIZE_BYTES } from 'src/const'
-import { Telegraf } from 'telegraf'
-import { PhotoSize, User } from 'telegraf/typings/core/types/typegram'
+import { idTag, USER_PICTURE_MAX_SIZE_BYTES } from 'src/const'
+import { Context, Markup, Telegraf } from 'telegraf'
+import { PhotoSize, Update, User } from 'telegraf/typings/core/types/typegram'
+import { Card, TelegramMessage } from './types'
+
+export async function ackMessage(message: TelegramMessage, ack: AckFunction) {
+  await ack({ tags: { [idTag]: `${message.message_id}` } })
+}
+
+export async function sendCard(payload: Card, client: Telegraf<Context<Update>>, chat: string, ack: AckFunction) {
```

Comment:
minor nitpick, but maybe we could put the `sendCard` function in a separate file as "misc.ts" sounds too generic for an important function like one for sending a card message? (at least when compared to how we have structured many other integrations)